File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/cadastro/usuario/UsuarioUnidadeNegocioDTO.java
package br.com.grupopipa.gestaointegrada.cadastro.usuario;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Getter
@Setter
@Builder
@AllArgsConstructor
public class UsuarioUnidadeNegocioDTO {
    private UUID unidadeNegocioId;
    private String unidadeNegocioNome;
    private Boolean isDefault;
}

File: src/frontend/src/app/components/cadastro/usuario/model/usuario-unidade-negocio-dto.ts
export class UsuarioUnidadeNegocioDTO {
  unidadeNegocioId: string;
  unidadeNegocioNome: string;
  unidadeNegocioCodigo: string;
  isDefault: boolean;

  constructor(
    unidadeNegocioId: string,
    unidadeNegocioNome: string,
    unidadeNegocioCodigo: string,
    isDefault: boolean
  ) {
    this.unidadeNegocioId = unidadeNegocioId;
    this.unidadeNegocioNome = unidadeNegocioNome;
    this.unidadeNegocioCodigo = unidadeNegocioCodigo;
    this.isDefault = isDefault;
  }
}
